src/pages/Tournament/TournamentDetails/BasicInfoPanel/parts.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Button, Modal } from 'antd';
+
+import { ClosableComponentProps } from 'utils/interfaces';
+import useFetch from 'hooks/useFetch';
+import useToastPushSubmit from 'hooks/useToastPushSubmit';
+import useEventCallback from 'hooks/useEventCallback';
+import { MessageResponse } from 'utils/apiResponseShapes';
```

Comment:
Codacy has a fix for the issue: [Insert `es';⏎import·{·ClosableComponentProps·}·from·'utils/interfac`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { MessageResponse } from 'utils/apiResponseShapes';
```